yellowbrick/features/pca.py

```diff
@@ -280,7 +290,7 @@ def pca_decomposition(X, y=None, ax=None, features=None, scale=True,
     visualizer = PCADecomposition(
         ax=ax, features=features, scale=scale, proj_dim=proj_dim,
         proj_features=proj_features, color=color, colormap=colormap,
-        random_state=random_state, **kwargs
+        alpha=alpha, random_state=random_state,**kwargs
```

Comment:
It looks like the spaces were modified here? We should reinsert the space after the last comma and before `**kwargs`